### PR TITLE
Automatic commit: Move 'cp-mobile-dev-kit-add-validation', 'cp-mobile…

### PR DESCRIPTION
**Pull request by jitendrakansal**
updated MDK tutorials with pre-requisite tutorial group and added reference to public github repo

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-upload-logs/cp-mobile-dev-kit-upload-logs.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ author_profile: https://github.com/jitendrakansal
 ---
 
 ## Details
+  - You may clone an existing project from [GitHub repository](https://github.com/SAP-samples/cloud-mdk-tutorial-samples/tree/master/2-Enhance-Your-First-MDK-App-with-Additional-Functionalities/3-cp-mobile-dev-kit-upload-logs) to start with this tutorial.
```
Comment (thecodester) [inline tutorials/cp-mobile-dev-kit-upload-logs/cp-mobile-dev-kit-upload-logs.md]: Skip a line after You will learn and move this text to there

File: tutorials/cp-mobile-dev-kit-online-app/cp-mobile-dev-kit-online-app.md [hunk 2]
```diff
@@ -10,12 +10,13 @@ author_profile: https://github.com/jitendrakansal
...
 
 ## Prerequisites
-- - **Tutorial:** [Set Up Initial Configuration for an MDK App](cp-mobile-dev-kit-ms-setup)
-- - **Tutorial:** [Enable Mobile Services App Development Tools](cp-mobile-dev-tools-setup)
+- **Tutorial group:** [Set Up for the Mobile Development Kit (MDK)](group.mobile-dev-kit-setup)
 - **Download and install** **SAP Mobile Services Client** on your [iOS](https://itunes.apple.com/us/app/sap-mobile-services-client/id1413653544?mt=8) or [Android](https://play.google.com/store/apps/details?id=com.sap.mobileservices.client) device
 - **Download and install** [Barcode Scanner](https://play.google.com/store/apps/details?id=com.google.zxing.client.android&hl=en) (required only for Android device)
 
 ## Details
+  - You may clone an existing project from [GitHub repository](https://github.com/SAP-samples/cloud-mdk-tutorial-samples/tree/master/3-Level-Up-with-the-Mobile-Development-Kit/1-Create-an-MDK-Online-App) and start directly with step 4 in this tutorial.
```
Comment (thecodester) [inline tutorials/cp-mobile-dev-kit-online-app/cp-mobile-dev-kit-online-app.md]: Move this text after You will learn

File: tutorials/cp-mobile-dev-kit-link-entity/cp-mobile-dev-kit-link-entity.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ author_profile: https://github.com/jitendrakansal
 ---
 
 ## Details
+  - You may clone an existing project from [GitHub repository](https://github.com/SAP-samples/cloud-mdk-tutorial-samples/tree/master/2-Enhance-Your-First-MDK-App-with-Additional-Functionalities/5-cp-mobile-dev-kit-link-entity) to start with this tutorial.
```
Comment (thecodester) [inline tutorials/cp-mobile-dev-kit-link-entity/cp-mobile-dev-kit-link-entity.md]: Move text after You will learn

File: tutorials/cp-mobile-dev-kit-error-archive/cp-mobile-dev-kit-error-archive.md [hunk 4]
```diff
@@ -10,12 +10,13 @@ author_profile: https://github.com/jitendrakansal
...
 
 ## Prerequisites
-- **Tutorial:** [Setup initial configuration for an MDK App in SAP Cloud Platform Mobile Services](cp-mobile-dev-kit-ms-setup)
-- **Tutorial:** [Enable Mobile Services App Development Tools](cp-mobile-dev-tools-setup)
+- **Tutorial group:** [Set Up for the Mobile Development Kit (MDK)](group.mobile-dev-kit-setup)
 - **Download and install:** **SAP Mobile Services Client** on your [iOS](https://itunes.apple.com/us/app/sap-mobile-services-client/id1413653544?mt=8) or [Android](https://play.google.com/store/apps/details?id=com.sap.mobileservices.client) device
 - **Download and install** [Barcode Scanner](https://play.google.com/store/apps/details?id=com.google.zxing.client.android&hl=en) (required only for Android device)
 
 ## Details
+  - You may clone an existing project from [GitHub repository](https://github.com/SAP-samples/cloud-mdk-tutorial-samples/tree/master/3-Level-Up-with-the-Mobile-Development-Kit/2-Handle-Error-Archive-in_an-MDK-App) and start directly with step 7 in this tutorial.
```
Comment (thecodester) [inline tutorials/cp-mobile-dev-kit-error-archive/cp-mobile-dev-kit-error-archive.md]: Move text after You will learn

File: tutorials/cp-mobile-dev-kit-create-customer/cp-mobile-dev-kit-create-customer.md [hunk 5]
```diff
@@ -10,10 +10,11 @@ author_profile: https://github.com/jitendrakansal
 ---
 
 ## Prerequisites
-- You have created an MDK app as described in [Get Started with the Mobile Development Kit](https://developers.sap.com/mission.mobile-dev-kit-get-started.html).
+- You have created an MDK app as described in [Create Your First Mobile App with the Mobile Development Kit (MDK)](mission.mobile-dev-kit-get-started).
 
 ## Details
-**Note**: Ensure that you have created an MDK app as described in [Get Started with the Mobile Development Kit](https://developers.sap.com/mission.mobile-dev-kit-get-started.html).
```
Comment (thecodester) [inline tutorials/cp-mobile-dev-kit-create-customer/cp-mobile-dev-kit-create-customer.md]: Move this and all extra text after  You will learn section

**Comment by jitendrakansal** [comment]
@thecodester i have updated tutorials with your suggestions. Please have a look.

File: tutorials/cp-mobile-dev-kit-create-customer/cp-mobile-dev-kit-create-customer.md (hunk 5 above)
Comment (MichaelCzcz) [reply tutorials/cp-mobile-dev-kit-create-customer/cp-mobile-dev-kit-create-customer.md]: I am working with Daniel.
Thanks for making the changes, everything seems great, but one more thing, please move the note also to after "you will learn".

Comment (jitendrakansal) [reply tutorials/cp-mobile-dev-kit-create-customer/cp-mobile-dev-kit-create-customer.md]: @MichaelCzcz i have removed the "note" part as this is already covered in the pre-requisite. Pls have a look and let me know if something is missing.

**Pull request merged**